modules/ismp/pallets/demo/src/lib.rs

```diff
@@ -349,9 +356,18 @@ impl<T: Config> IsmpModule for IsmpModuleCallback<T> {
 			Response::Post(_) => Err(IsmpError::Custom(
 				"Balance transfer protocol does not accept post responses".to_string(),
 			))?,
-			Response::Get(res) => Pallet::<T>::deposit_event(Event::<T>::GetResponse(
-				res.values.into_values().collect(),
-			)),
+			Response::Get(res) => {
+				res.values.iter().for_each(|(k, v)| {
+					if v.is_some() {
+						let hash_key = Twox128::hash(k.as_ref());
+						GetResponses::<T>::insert(hash_key, v)
+					}
+				});
```

Comment:
Why?

Reply:
David advised me to put the returned value in the storage and just read the storage

Reply:
why?

Reply:
So as not to track the event and just fetch the value

Reply:
use wait_for_finalized